packages/extension/src/ui/features/accountNfts/useNFTCollections.ts

```diff
@@ -22,6 +22,7 @@ export const useCollectionAndNFTs = (
     () => fetchCollectionAndNfts(nonFungibleTokenIds, network),
     {
       ...config,
+      dedupingInterval: 5000,
```

Comment:
Is this change necessary from 2 seconds to 5 seconds?

In what cases, will the wallet try to query the same data within a few seconds?

Reply:
When I click around in the extension wallet as a user while monitoring the network tab (e.g. switching tabs back and forth, etc), it feels that 2 seconds is still causing unnecessary calls. 

Not sure what's the best number to set here but feels that 5 seconds might be a good spot between timely update and reducing unnecessary calls.  WDYT?

Reply:
If all of the token&NFT data is cached, there should be no API calls. Why there will be extra calls when clicking around?

Reply:
There are two calls that are not cached, where we have adjusted `dedupingInterval` for:

1. The call to get account balance
2. For NFTs, we have a call to fetch metadata from the explorer backend in batch [here](https://github.com/alephium/extension-wallet/blob/fix-display-nft/packages/extension/src/ui/features/accountNfts/alephium-nft.service.ts#L16-L20), since the payload of the call is dynamic.

Reply:
So for NFT, we will need to re-fetch metadata every time we click on the NFT screen?

Reply:
Yeah, we used to cache them when we fetch the NFT metadata individually, after migrating to fetching the metadata in batch from explorer backend, it is not cached any more due to its dynamic payload.

I will try to cache them.